.changeset/brown-bananas-repair.md

```diff
@@ -0,0 +1,5 @@
+---
+"skuba": minor
+---
+
+template/koa-rest-api: Add opentelemetry support
```

Comment:
```suggestion
template/koa-rest-api: Add opt-in OpenTelemetry support
```

Q: should we add this to the Express template too?

Reply:
Sure, do you want me to do it in the same PR or separate?

Reply:
Feel free to merge this one in first if that's easier